Unable to run generated app - build with error: 'certFilename' does not exist in the current context

Unable to run generated app using the latest version.
 
Program.cs(82,165): error CS0103: The name 'certFilename' does not exist in the current context 
 Program.cs(82,193): error CS0103: The name 'CertValidationCallback' does not exist in the current context TestDB.csproj

See last commit https://github.com/jhipster/jhipster-dotnetcore/commit/54cd2b3ab11f8856d7f411a36b17d8e057202889

--- a/generators/server/templates/dotnetcore/src/Project/Program.cs.ejs
+++ b/generators/server/templates/dotnetcore/src/Project/Program.cs.ejs
@@ -93,7 +93,7 @@ namespace <%= namespace %> {
                 : "JhipsterApp";
             var loggerConfiguration = new LoggerConfiguration()
                 .Enrich.With<LoggerNameEnricher>()
-                .WriteTo.TcpSyslog(url,port,appName, FramingType.OCTET_COUNTING,SyslogFormat.RFC5424,Facility.Local0,SslProtocols.None, new CertificateFileProvider(certFilename, String.Empty),CertValidationCallback)
+                .WriteTo.TcpSyslog(url,port,appName, FramingType.OCTET_COUNTING,SyslogFormat.RFC5424,Facility.Local0,SslProtocols.None)
                 .ReadFrom.Configuration(appConfiguration);
 
             return loggerConfiguration.CreateLogger();

